build/providers/admob/android/config.json

```diff
@@ -1,6 +1,6 @@
 {
  "jars": [
-    "admobadapter-3.1.1.jar",
+    "admobadapter-4.0.3.jar",
```

Comment:
where did you get this file ?

Reply:
this file is from ironsource itself. initially was going in wrong direction. @rampr